code/config/lib/Loot/Config/Yaml.hs

```diff
@@ -3,24 +3,27 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
 {-# LANGUAGE DataKinds            #-}
-{-# LANGUAGE KindSignatures       #-}
+{-# LANGUAGE TypeFamilies         #-}
 {-# LANGUAGE TypeOperators        #-}
 {-# LANGUAGE UndecidableInstances #-}
-{-# LANGUAGE MonoLocalBinds       #-}
 
 -- | Utilities for reading configuration from a file.
 module Loot.Config.Yaml
        ( configParser
        ) where
 
-import Data.Aeson (FromJSON (parseJSON))
+import Data.Aeson (FromJSON(parseJSON), ToJSON(toJSON), Value(Object))
 import Data.Aeson.BetterErrors (Parse, fromAesonParser, keyMay, keyOrDefault, toAesonParser')
-import Data.Vinyl (Rec ((:&), RNil))
+import Data.Aeson.Types (Object)
+import Data.Vinyl (Rec(RNil, (:&)))
 import GHC.TypeLits (KnownSymbol, symbolVal)
 
-import Loot.Config.Record ((:::), (::<), (::+), (::-), ConfigKind (Partial),
-                           ConfigRec, Item (ItemOptionP, ItemSub, ItemSumP, ItemBranchP),
-                           ItemKind, SumSelection)
+import qualified Data.HashMap.Strict as HM
+
+import Loot.Config.Record
+  (ConfigKind(Final, Partial), ConfigRec,
+  Item(ItemBranchF, ItemBranchP, ItemOptionF, ItemOptionP, ItemSub, ItemSumF, ItemSumP), ItemKind,
+  SumSelection, (::+), (::-), (:::), (::<))
```

Comment:
I like this style of imports with long list of items (when there are just 2 spaces) more, but I see that in the previous version this list of imported items started on the same line as `import` keyword and subsequent lines were aligned with long indentation. Is it possible to configure `stylish-haskell` to match the existing style?

Reply:
I think it's possible, maybe someone have this config locally, I'll try to do it

Reply:
my hls is mad... yesterday it worked well and now it always loading smth, and formatter stopped working (hls provide stylish-haskell)... so I returned formatting by hand

Reply:
I was told that sometimes it may help to delete `~/.cache/hie-bios` or `~/.cache/ghcide` or something like that (if you have it) to clear cache and fix HLS.

Reply:
Didn't helped(
<img width="483" alt="image" src="https://user-images.githubusercontent.com/6579275/126486421-feecc03b-8cfd-443e-af45-fded5c6ccc1b.png">

that's all